Source/Scene/Cesium3DTilesetTraversal.js

```diff
@@ -741,6 +735,17 @@ define([
         return visibilityPlaneMask !== CullingVolume.MASK_OUTSIDE;
     }
 
+    function isVisibleAndMeetsSSE(tileset, tile, frameState) {
+        var maximumScreenSpaceError = tileset._maximumScreenSpaceError;
+        var parent = tile.parent;
+        if (!defined(parent)) {
+            return isVisible(tile._visibilityPlaneMask);
+        }
+        var showAdditive = parent.refine === Cesium3DTileRefine.ADD && parent._screenSpaceError > maximumScreenSpaceError;
+
+        return isVisible(tile._visibilityPlaneMask) && (!showAdditive || getScreenSpaceError(tileset, parent.geometricError, tile, frameState) > maximumScreenSpaceError);
```

Comment:
You may want to check this, but I believe that by the time you call this function in `loadTile`, the results of `getScreenSpaceError` have already been cached in `tile._screenSpaceError` which gets computed in `visitStart`.

Reply:
`computeSSE` caches but `getScreenSpaceError` doesn't, so this should be ok.